js/Notification.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+class Notification {
+    constructor(options) {
+        this._el = document.createElement('div');
+        this._el.className = "notification";
+        this._el.innerHTML = options.text;
+        if (options.addClass) {
+            this._el.classList.add(options.addClass);
```

Comment:
It is an additional style class. It changes some style of the Notification. I've renamed it to additionalStyleClass for more transparency.